docs/source/installation_launch.md

```diff
@@ -1,6 +1,10 @@
-# Launching with Docker
+# Getting Started
```

Comment:
Let's maybe call this Quicktour (and `quicktour` ) for consistency with other libraries

* https://huggingface.co/docs/accelerate/basic_tutorials/install
* https://huggingface.co/docs/datasets/v2.14.4/quickstart
* https://huggingface.co/docs/transformers/quicktour
* https://huggingface.co/docs/huggingface_hub/quick-start (this one is different but same idea)